src/index.js

```diff
@@ -10,10 +10,23 @@ const addEventsUrl = baseUrl + '/addEvents';
 const uploadLogsUrl = baseUrl + '/api/uploadLogs';
 
 let logGroupOptions = {};
-if (process.env['LOG_GROUP_OPTIONS']) logGroupOptions = JSON.parse(process.env['LOG_GROUP_OPTIONS']);
-let defaultParserName = process.env['PARSER_NAME'];
+let userLogFile = null;
+let userParserName = null;
+let userServerHost = null;
+if (process.env['LOG_GROUP_OPTIONS']) {
+    logGroupOptions = JSON.parse(process.env['LOG_GROUP_OPTIONS']);
+    let functionLogGroupName = process.env['AWS_LAMBDA_LOG_GROUP_NAME'].substr(1);
+    if (logGroupOptions[functionLogGroupName]) {
```

Comment:
"logGroupOptions[functionLogGroupName]" is repeated on this line and the next 4 - probably cleaner to assign it to a variable before the if statement